### PR TITLE
Added VOICE_BUNDLE productType and VOUCHER_STATUS_UNKNOWN errorType

### PR DESCRIPTION
**Pull request by simone-van-zyl**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
     <swagger-maven-plugin-version>3.1.3</swagger-maven-plugin-version>
     <logback-version>1.0.13</logback-version>
     <major-version>5</major-version>
-    <minor-version>18</minor-version>
+    <minor-version>19</minor-version>
```
Comment (RichardFourie) [inline pom.xml:45]: Don't forget about the version number in the readme.

File: src/docs/devguide/hugo/site/content/release-notes.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 This page describes changes to the Airtime Service Interface implemented across different releases of the interface.
 
+## v5.19.0
+Released 14 October 2020
+
+- Added new `ProductType`, `VOICE_BUNDLE`,  and new `ErrorType`, `VOUCHER_STATUS_UNKNOWN`.
+
 ## v5.18.0
 
 Released TBD
```
Comment (RichardFourie) [inline src/docs/devguide/hugo/site/content/release-notes.md:10]: Seems like someone forgot to update the date for the previous version.

File: pom.xml (hunk 1 above)
Comment (simone-van-zyl) [reply pom.xml:45]: Changed the readme =)

File: src/docs/devguide/hugo/site/content/release-notes.md (hunk 2 above)
Comment (simone-van-zyl) [reply src/docs/devguide/hugo/site/content/release-notes.md]: Fixed it according to git commit date for final release of v5.18.0

File: src/main/java/io/electrum/airtime/api/model/ErrorDetail.java [hunk 3]
```diff
@@ -46,7 +46,7 @@
       CARD_EXPIRED("CARD_EXPIRED"),
       INCORRECT_PIN("INCORRECT_PIN"),
       PIN_ATTEMPTS_EXCEEDED("PIN_ATTEMPTS_EXCEEDED"),
-      VOUCHER_STATUS_UNKNOWN("VOUCHER_STATUS_UNKNOWN");
+      OUTCOME_UNKNOWN("OUTCOME_UNKNOWN");
```
Comment (simone-van-zyl) [inline src/main/java/io/electrum/airtime/api/model/ErrorDetail.java:49]: Changed after discussion with @TimElectrum

**Review by RichardFourie — APPROVED** [review]
Remember to update the date and then you can release.

**Review by TimElectrum — CHANGES_REQUESTED** [review]
please also update the `ordinalDataProvider` data provider in the `io.electrum.airtime.api.NewModelTest`

File: src/docs/devguide/hugo/site/content/release-notes.md [hunk 4]
```diff
@@ -1,8 +1,13 @@
 This page describes changes to the Airtime Service Interface implemented across different releases of the interface.
 
+## v5.19.0
+Released 14 October 2020
+
+- Added new `ProductType`, `VOICE_BUNDLE`,  and new `ErrorType`, `OUTCOME_UNKNOWN`.
```
Comment (TimElectrum) [inline src/docs/devguide/hugo/site/content/release-notes.md:6]: also, the rapidly changing emphasis/spacing/font for each word in this sentence makes my eyes go weird. And one bullet point makes it look like you did one thing. Hows about:
```suggestion
- Added a new `ProductType` called `VOICE_BUNDLE`.
- Added a new `ErrorType` called `OUTCOME_UNKNOWN`.
```

**Review by TimElectrum — APPROVED** [review]

**Pull request merged**